pkgs/by-name/pa/pa-notify/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, glib
+, libnotify
+, libpulseaudio
+, pkg-config
+, util-linux
+,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "pa-notify";
+  version = "1.5.0";
+
+  src = fetchFromGitHub {
+    owner = "ikrivosheev";
+    repo = "pa-notify";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-356qwSxxxAUNJajsVjH3zqGAZQwMOcoLPSKPZdsCmBM=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    extra-cmake-modules
+    glib.dev
+    libnotify
+    libpulseaudio
+    pkg-config
+    util-linux.dev
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/ikrivosheev/pa-notify";
+    description = "PulseAudio or PipeWire volume notification";
+    license = licenses.mit;
+    maintainers = with maintainers; [ juancmuller ];
+    mainProgram = "pa-notify";
+    platforms = platforms.unix;
```

```suggestion
    platforms = platforms.linux;
```
Are PulseAudio or PipeWire a thing outside of Linux?

---

Both work on BSD. Happy to change.